From bb94c4f2b5ed50856b712c20e5bf363d223a8b41 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Mon, 17 Dec 2018 20:38:41 +0100
Subject: [PATCH 032/486] media: sun6i-csi: Add support for missing 16bit color
 formats

CSI engine will not try to reorder the color parts, so this is safe
for matching mbus formats.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/media/platform/sunxi/sun6i-csi/sun6i_csi.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/media/platform/sunxi/sun6i-csi/sun6i_csi.c b/drivers/media/platform/sunxi/sun6i-csi/sun6i_csi.c
index 0cc99e3fd..2c0a5c632 100644
--- a/drivers/media/platform/sunxi/sun6i-csi/sun6i_csi.c
+++ b/drivers/media/platform/sunxi/sun6i-csi/sun6i_csi.c
@@ -360,6 +360,9 @@ static enum csi_input_seq get_csi_input_seq(struct sun6i_csi_dev *sdev,
 		break;
 
 	case V4L2_PIX_FMT_YUYV:
+	case V4L2_PIX_FMT_YVYU:
+	case V4L2_PIX_FMT_UYVY:
+	case V4L2_PIX_FMT_VYUY:
 		return CSI_INPUT_SEQ_YUYV;
 
 	default:
-- 
2.35.3

